src/helpers.ts

```diff
@@ -79,7 +79,10 @@ export const checkPackageUpdates = async (
   try {
     const pkgName = packageName ?? name
     const response = await request<{ version: string }>(
-      `https://registry.npmjs.org/${pkgName}/latest`
+      `https://registry.npmjs.org/${pkgName}/latest`,
+      undefined,
+      1,
+      true
```

Comment:
One more downside of the arguments nature of `request`. I also meant this when I suggested converting `request` arguments to an extended options object to avoid having `undefined` for skipped args. 🙂

Reply:
True, this doesn't look good overall as well.

I've made a simple change in the structure which would accommodate all requirements while keeping `undefined` away but the types are a bit assertive.

```
interface ExtendedRequestInit extends RequestInit {
  retries?: number
  skipTrackingHeaders?: boolean
}

export const request = async <T = Response>(
  url: RequestInfo | URL,
  options = {
    retries: (this as unknown as ExtendedRequestInit)?.retries ?? 1,
    skipTrackingHeaders:
      (this as unknown as ExtendedRequestInit)?.skipTrackingHeaders ?? false,
  } as ExtendedRequestInit
): Promise<T> => {
```

Reply:
# F.M.L.

TIL we can give typed default args this way 😭 
Thanks a lot @chybisov